src/storage/v3/shard_rsm.cpp

```diff
@@ -457,6 +492,129 @@ EdgeFiller InitializeEdgeFillerFunction(const msgs::ExpandOneRequest &req) {
   return edge_filler;
 }
 
+msgs::ReadResponses HandleReadWithOrderBy(msgs::ExpandOneRequest &&req, Shard::Accessor &&acc) {
```

Comment:
I don't like the idea of handling the requests separately depending on whether it contains an order by or not. E.g.: this version doesn't take into account (at least I cannot spot where) the `src_vertices` field of the request which is essential for this request.